### ISSUE TITLE
i18n script doesn't automatically create pluralized strings

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#13061]
It just creates the "other" string. It should probably create the singular too.

**Comment by dbkr** [comment]
I think this might require teaching the script how to write English though? It could probably error out until you add the other forms though.

**Comment by turt2live** [comment]
We already assume it is English? It only ever writes to `en_EN.json`

**Comment by turt2live** [comment]
oh, but it won't know what the singular form is... Maybe we can make `_t()` take singular forms too?

**Comment by t3chguy** [comment]
Right, but we'd have to teach it to guess which word to pluralise.
